COLLABORATOR_GUIDE.md

```diff
@@ -83,7 +83,7 @@ The Website also uses several other Open Source libraries (not limited to) liste
 - [Shiki][] is a Syntax Highlighter used for our Codeboxes
   - The syntax highlighting is done within the processing of the Markdown files with the MDX compiler as a Rehype plugin.
 - [MDX][] and Markdown are used for structuring the Content of the Website
-- [`next-intl`][] is the i18n Library adopted within the Website
+- \[`next-intl`]\[] is the i18n Library adopted within the Website
```

Comment:
Why `\`?

Reply:
Good question, 'cause I didn't add it manually. It's can from Prettier ?